content/cecil-starter-blog-with-netlify-cms.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Cecil starter blog (with Netlify CMS)"
+github: https://github.com/Cecilapp/starter-blog
+author: Cecilapp
+demo: https://demo.cecil.app
+date: 2020-06-03
+ssg:
+    _ Cecil
```

Comment:
Cecil will need to be added as a new ssg taxonomy. Also the syntax here is wrong and crashing the build should be a hypen instead of an underscore

Reply:
Fixed